DigitalRain/Raindrops/IRaindrop.cs

```diff
@@ -6,15 +6,10 @@
 
 namespace DigitalRain.Raindrops
 {
-    interface IRaindrop
-    {
-        char[] SymbolPool { get; }
-        char Symbol { get; }
-        double LifeRemaining { get; }
-        static Color DefaultColor { get; }
+    using GameUtilities;
 
-        bool IsDead();
-        string SymbolAsStr();
-        char GetSymbolFromPool();
+    public interface IRaindrop : IGameObject
```

Comment:
I guess I'm ok with this simplification. I figured the interface would evolve organically. So I appreciate you taking the liberty of gutting it.

It's nice to be able to touch and feel our way though the code instead of assume every file can never be changed lol

Reply:
Yeah none of this stuff was used by any client of a raindrop.  It's easier to add stuff to an interface than to take it away so I wanted to yank it out now and then add it back later if really needed. This also gives you a clearer picture of what makes a raindrop a raindrop (something I needed to know when implementing the glitched one).